frontend/src/Explorer/View/Common.purs

```diff
@@ -48,6 +59,26 @@ transactionHeaderView (CTxEntry entry) =
               ]
           ]
 
+transactionHeaderView' :: CTxSummary -> P.Html Action
+transactionHeaderView' (CTxSummary txSummary) =
+    P.div
+          [ P.className "transaction-header"]
+          [ P.div
+              [ P.className "hash" ]
```

Comment:
Minor. I would recommend having Class.purs where we defined these class name strings and import and use them from there. I guess there are reused in other views as well